crates/trie/parallel/src/proof_task.rs

```diff
@@ -77,1255 +77,1290 @@ use std::{
     time::{Duration, Instant},
 };
 use tokio::runtime::Handle;
-use tracing::{debug, debug_span, error, trace};
+use tracing::{debug_span, error, trace};
 
 #[cfg(feature = "metrics")]
 use crate::proof_task_metrics::ProofTaskTrieMetrics;
 
 type StorageProofResult = Result<DecodedStorageMultiProof, ParallelStateRootError>;
 type TrieNodeProviderResult = Result<Option<RevealedNode>, SparseTrieError>;
-type AccountMultiproofResult =
```

Comment:
removed `AccountMultiproofResult` as it is redundant. Both account and storage results now come as `ProofResult`, with the account case carried by `ProofResult::AccountMultiproof(multiproof, stats)`.